spec/models/video_spec.rb

```diff
@@ -1,43 +1,35 @@
 require 'spec_helper'
 
 describe Video do
+  it { should belong_to(:category) }
+  it { should validate_presence_of(:title) }
+  it { should validate_presence_of(:description) }
+
+  let(:first_video) { Fabricate(:video, title: "The first title", description: "The first description", created_at: 1.day.ago) }
+  let(:second_video) { Fabricate(:video, title: "The second title", description: "The second description") }
+
   it "saves itself" do
-    first_video = Video.create(title: "The first title", description: "The first description")
     first_video.save
     expect(Video.first).to eq(first_video)
   end
 
-  it { should belong_to(:category) }
-  it { should validate_presence_of(:title) }
-  it { should validate_presence_of(:description) }
-
   it "returns search by title by title" do
-    first_video = Video.create(title: "The first title", description: "The first description")
-    second_video = Video.create(title: "The second title", description: "The second description")
     expect(Video.search_by_title("hello")).to eq([])
   end
 
   it "should return a single item on a partial match" do
-    first_video = Video.create(title: "The first title", description: "The first description")
-    second_video = Video.create(title: "The second title", description: "The second description")
     expect(Video.search_by_title("first")).to eq([first_video])
```

Comment:
Same as above, here you create only 1 video record so it will most likely find that one - the spec does not really show it.